src/index.ts

```diff
@@ -0,0 +1,102 @@
+export const booleanCalculator = (input:string):boolean | void => {
+    let evaluatedStatement = [];
+    input = formatInput(input);
+    
+    let inputArray: string[] = input.split(" ");
+    evaluatedStatement = evaluator(inputArray);
+    if (input == "NOT TRUE" || input == "FALSE") {
+        return false;
+    } else if (input == "NOT FALSE" || input == "TRUE") {
+        return true;
+    } else if (evaluatedStatement[0] == "TRUE") {
+        return true;
+    } else if (evaluatedStatement[0] == "FALSE") {
+        return false;
+    }
+}
+
+const formatInput = (input:string):string => {
+    let formattedInput:string[] = Array.from(input);
+    for (let i=0; i<formattedInput.length; i++) {
+        if (formattedInput[i] == '(') {
+          formattedInput.splice(i + 1,0, ' ');
+          break;
+        }
+    }
+    for (let j=0; j<formattedInput.length; j++) {
+        if (formattedInput[j] == ')') {
+          formattedInput.splice(j,0,' ');
+          break;
+        }
+    }
+    return formattedInput.join('');
+
+}
+
+const evaluator = (statementArray:string[]):string[] => {
+    let parenthesesEvaluated = evaluateParentheses(statementArray)
+    let notEvalulated = evaluateNot(parenthesesEvaluated);
+    let andEvaluated = evaluateAnd(notEvalulated);
+    let orEvaluated = evaluateOr(andEvaluated);
+    if (statementArray.length > 1) {
```

Comment:
I think this line should be redundant. If its required, I think it would be better if it was redundant